app/src/debug/AndroidManifest.xml

```diff
@@ -43,12 +43,12 @@
         <activity
             android:name=".signup.SignUpActivity"
             android:theme="@style/SignUp_Theme"
-            android:windowSoftInputMode="stateVisible|adjustResize" />
```

Comment:
Why is this changed?

Reply:
Before changing this, the keyboard would automatically popup (with focus on the first EditText), without the user doing anything, and move all UI elements upwards.